ingest/api/ingestapi.py

```diff
@@ -309,8 +318,21 @@ def get_related_entities_count(self, relation, entity, entity_type):
                 return page.get('totalElements')
             return len(result["_embedded"][entity_type])
 
-    def create_project(self, submission_url, content, uuid=None):
-        return self.create_entity(submission_url, {'content': content}, "projects", uuid)
+    def create_project(self, submission_url: str, content: dict, uuid: str = None, token: str = None):
+        if not submission_url:
+            project = self._create_project_without_submission(content, token)
+        else:
+            project = self.create_entity(submission_url, {'content': content}, "projects", uuid)
+        return project
+
+    def _create_project_without_submission(self, content, token):
+        headers = dict.copy(self.get_headers())
```

Comment:
Could you tell me why we need the `dict.copy()` operation, please? Tx

Reply:
The `get_headers` method returns the default headers, i just dont want to set the token in the default headers if it doesn't need to.